app/views/ConfirmationPageView.scala.html

```diff
@@ -76,5 +77,24 @@
         "cf.historic.document.request.confirmation.back-to-dashboard.link-text"),
         location = returnLink,
         pId = Some("link-text"),
-        pClass = "govuk-body govuk-!-margin-bottom-9")
+    )
+        
+    @h2(
+        classes="govuk-heading-s",
+        id=Some("email-confirmation-subheader-help"),
```

Comment:
Better to keep ids something related to (improve the service)  like 'improve-the-service-heading' and similarly at line 90